### DISCUSSION TITLE
🥘Lunch de la team domaine-nc 🧑‍🤝‍🧑

### DISCUSSION THREAD

**Discussion by adriens** [adriens/domaine-nc-javafx#45] [Polls]
# :grey_question: A propos

Ce projet a été rendu possible car : 

- :heavy_check_mark:  Un SDK a été développé
-  :heavy_check_mark: Une API a été développée et pakcagée en image Docker
-  :heavy_check_mark: Déployée en continu
-  :heavy_check_mark: Publiée sur un marketplace d'API
-  :heavy_check_mark: Un dev mobile inspirant a été réalisé
-  :heavy_check_mark:  Des étudiants on choisi ce projet

Aujourd'hui, je souhaite que toute la team se rencontrer autour d'un déjeuner 😋 afin que : 

-  Nous célébriions ensemble ce succès
- Que j vous présente un nouveau membre de l'équipe et les perspectives prochaines avec vous

# :dart: Actions

1. :ballot_box:  Voter (ça c'est vous la team)
2. :calendar:  Organiser le lunch (@adriens)
3. :camera_flash: Immortaliser

# :tickets: Issues liées

- https://github.com/adriens/domaine-nc-javafx/issues/43
- https://github.com/adriens/domaine-nc-javafx/issues/39
- https://github.com/adriens/domaine-nc-javafx/issues/44
- https://github.com/adriens/domaine-nc-javafx/issues/46

**Comment by adriens** [comment]
:point_right: @Kevin-Petit  @IsoerNC  @Davnox900

**Comment by MelanieGault** [comment]
Mes dispo : 
* mercredi prochain (donc le 14 juin)
* puis les 28 29 ou 30 juin.

**Comment by lschaeffer313** [comment]
Je ne pourrais pas la semaine du 12 juin par contre semaine prochaine je pourrais.

**Comment by kvnpetit** [comment]
@Davnox900 @IsoerNC nous somme actuellement disponible la semaine prochaine et semaine d'après.

**Reply by adriens** [reply]
@Kevin-Petit c'est @lschaeffer313 qui a créé le design dark back/yellow :smile_cat: :sunglasses: :nerd_face:

**Comment by adriens** [comment]
Good, je vais lancer un calendly dédié et zou :100:

**Comment by lschaeffer313** [comment]
Je rectifie je serais dispo à partir de mercredi de la semaine prochaine 🚀

**Reply by adriens** [reply]
Je crois que ça nous faire un créneau ça :rocket:

**Comment by adriens** [comment]
[:calendar: DOODLE](https://doodle.com/meeting/participate/id/aQ7176Ld)

**Comment by adriens** [comment]
EH EH 
![image](https://github.com/adriens/domaine-nc-javafx/assets/5235127/4ec0aac7-d867-484c-ae1a-5ca898b9438c)

**Comment by adriens** [comment]
Voila, je suis reparti sur une planficication qui tente de prendre en compte au mieux les dispos de chacun : 

> Suite à de nombreux succès autour de l'API publique des noms de domaine en Nouvelle-Calédonie, je que nous partagions (en présenciel :-)) un déjeuner afin que tout un chacun puisse rencontrer l'équipe au complet.
> 
> Comme vous pourrez le voir, nous aurons parmi nous : 
> 
> - Développeurs : Michèle, Spencer (OPT)
> - API Lifecycle management et intégrateur RapidAPI : Pipiena (ATOS)
> - Managers : Sabrina (OPT)
> - Développeur mobile Flutter : Laurent (DEVEO)
> - Designer Logo et testeur : Christophe (Calinfo IT)
> - Ronny  : testeur et idéation(/style UX app mobile (Skazy)
> - Dévs Java FX : Kevin, David, Irwan (Etudiants UNC)
> - Intégrateur API Marketplace APIGEE : Mélanie (Stratégie Zen IT)
> - Zoe : plateforme Testeum, (HIGHTEST)
> 
>

**Comment by adriens** [comment]
![image](https://github.com/adriens/domaine-nc-javafx/assets/5235127/13dac774-5bd9-41e1-8d26-c5a37ee8ac7d)

**Reply by spencer-forrest-ms-cs** [reply]
No chosen date yet ?

**Comment by adriens** [comment]
Still optiminz dates to get as much people as possible, please check the latest planning : I should be able to fix the final date very (very) soon this week

![image](https://github.com/adriens/domaine-nc-javafx/assets/5235127/dd27631d-0e61-4293-b3ee-4650851f4160)


:information_source: Also I've added Thursday to create more opportunites for all of us :crossed_fingers:

**Comment by adriens** [comment]
:+1: C booké !
![image](https://github.com/adriens/domaine-nc-javafx/assets/5235127/affe0563-c09a-4fb6-8e44-ce19135eff81)
![image](https://github.com/adriens/domaine-nc-javafx/assets/5235127/0ae4a2b9-4a70-466b-a6fd-448483576947)

**Comment by adriens** [comment]
![image](https://github.com/adriens/domaine-nc-javafx/assets/5235127/692b16c2-c106-4d8a-bbdf-4ed917ca675c)

**Comment by adriens** [comment]
C good c réservé :heavy_heart_exclamation: :hugs:

**Comment by adriens** [comment]
C today :muscle: :shallow_pan_of_food: :people_holding_hands:

**Comment by adriens** [comment]
https://www.tripadvisor.fr/Restaurant_Review-g294130-d4788999-Reviews-Snack_Ulysse-Noumea_Grand_Terre.html

**Comment by adriens** [comment]
[linkedIn](https://www.linkedin.com/posts/adrien-sales_apis-marketplace-collaboration-activity-7090105087166963712-cA7_?utm_source=share&utm_medium=member_desktop)

![image](https://github.com/opt-nc/domaine-nc-javafx/assets/5235127/0d58cde5-d1d3-418e-874f-588d792d94f3)

**Comment by adriens** [comment]
[Tweet](https://twitter.com/rastadidi/status/1684339902744510464)

![image](https://github.com/opt-nc/domaine-nc-javafx/assets/5235127/dc65319a-c329-4499-b016-c7e300d482a6)
